src/js/pensions/config/form.js

```diff
@@ -1385,13 +1374,7 @@ const formConfig = {
               'ui:options': {
                 expandUnder: 'view:spouseHasOtherExpenses'
               }
-            }),
-            'view:spouseOtherExpensesWarning': {
```

Comment:
@jbalboni should we also remove the accompanying schema property for this (lines 1372-1375)?